projects/igniteui-angular-i18n/README.md

```diff
@@ -2,9 +2,13 @@
 
 ## Purpose
 
-This package stores resource strings for the Ignite UI for Angular `i18n`. The English resource strings and the interfaces for each localizable component's `resourceStrings` property are stored in the [Ignite UI for Angular](https://github.com/IgniteUI/igniteui-angular) repository. The interfaces can be used as a schema for creating custom resource strings.
+This package stores resource strings for the Ignite UI for Angular `i18n`. The English resource strings and the interfaces for each localizable component's `resourceStrings` property are stored in the Ignite UI for Angular [project](https://github.com/IgniteUI/igniteui-angular) / [package](https://www.npmjs.com/package/igniteui-angular). The interfaces can be used as a schema for creating custom resource strings.
 
 ## File Usage
 
 An npm package will be produced and it could be referenced alongside with the igniteui-angular in the clients' projects.
 
+[Localization (18n)](https://www.infragistics.com/products/ignite-ui-angular/angular/components/general/localization) sample can be found alongside Ignite UI for Angular samples.
+
+## Old location
+This package used to be under the [igniteui-angular-i18n](https://github.com/IgniteUI/igniteui-angular-i18n) repository. It will no longer be used and maintained. All new feature and development will be under it's [new location](https://github.com/IgniteUI/igniteui-angular).
```

Comment:
Yeah that's a bit much for a footnote, especially the 'no longer used and maintained' part can give the wrong impression - the package is very much maintained, just the source moved. So either a note:
```suggestion
> Note: The source for the package has been moved to this repository from [igniteui-angular-i18n](https://github.com/IgniteUI/igniteui-angular-i18n) which will remain archived for history reference.
```